n/ngs-utils/ngs-utils-23.04.1.eb

```diff
@@ -0,0 +1,31 @@
+name = 'ngs-utils'
+version = '23.04.1'
+
+homepage = 'https://github.com/molgenis/ngs-utils'
+description = """Collection of notes and scripts related to NGS """
+
+toolchain = {'name': 'system', 'version': ''}
```

Comment:
```suggestion
toolchain = SYSTEM
```

Reply:
Both will work for now, but the latter is the future way to go.

Reply:
okay, check